packages/angular/src/utils/mf/utils.ts

```diff
@@ -48,7 +48,14 @@ export function getFunctionDeterminateRemoteUrl(isServer: boolean = false) {
   const remoteEntry = isServer ? 'server/remoteEntry.js' : 'remoteEntry.mjs';
 
   return function (remote: string) {
-    const remoteConfiguration = readCachedProjectConfiguration(remote);
+    let remoteConfiguration = null;
+    try {
+      remoteConfiguration = readCachedProjectConfiguration(remote);
+    } catch (e) {
+      throw new Error(
+        `Cannot find remote "${remote}". Check that the remote name is correct in your module federation config file.\n`
+      );
+    }
```

Comment:
Its ok to add a bespoke error message here, but can we also update the error messaging in `readCachedProjectConfiguration` s.t. it doens't look so bad in the first place?